### PR TITLE
Add seller to order action.

### PR DESCRIPTION
**Pull request by morloderex**
Remove order route from laracom. Sense siesta has the seller laracom only holds the action api to create order no route for storing them.

- Moved the api tests to unit tests around the action.

- I have added the person who made the order.

### CODE REVIEW

File: src/Orders/OrderSeller.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Laracommerce\Core\Orders;
+
+interface OrderSeller
```
Comment (viezel) [inline src/Orders/OrderSeller.php]: Rename to `OrderSalesRepresentative`

File: tests/Unit/Orders/OrderUnitTest.php [hunk 2]
```diff
@@ -169,4 +174,137 @@ public function it_can_fetch_from_the_container()
 
         $this->assertInstanceOf(OrderRepository::class, $interface);
     }
+
+    /** @test */
+    public function it_can_create_an_order_through_the_action_without_stock_lookup()
```
Comment (viezel) [inline tests/Unit/Orders/OrderUnitTest.php]: `..._without_stock_lookup`  to `_without_stock_management`

File: src/Orders/Factory/OrderValidationRules.php [hunk 3]
```diff
@@ -46,7 +47,10 @@ public static function internalRules(): array
             'order_status_id' => ['nullable', 'integer', Rule::exists('ecom_order_statuses', 'id')],
             'courier_id' => ['nullable', 'integer', Rule::exists('ecom_couriers', 'id')],
             'address_id' => ['nullable', 'integer', Rule::exists('ecom_addresses', 'id')],
-            'items_total' => ['required', 'integer', 'min:0']
+            'items_total' => ['required', 'integer', 'min:0'],
+            'seller_id' => ['nullable', 'integer'],
```
Comment (viezel) [inline src/Orders/Factory/OrderValidationRules.php]: refactor to `sales_rep_id`

Comment (viezel) [reply src/Orders/Factory/OrderValidationRules.php]: remove the seller name and seller email. This is not be provided by the API, but instead by the id internally with a DB lookup .

Comment (morloderex) [reply src/Orders/Factory/OrderValidationRules.php]: This is not validation rules for the api it's validation rules for importing or manually creating an order etc. It matches up with the database structure. seller_name and seller_email will never go through the api. Only the id of the sales rep user.

Comment (viezel) [reply src/Orders/Factory/OrderValidationRules.php]: yes, but refactor all naming from `seller` to `sales_rep`

File: src/Orders/OrderSalesRepresentative.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+namespace Laracommerce\Core\Orders;
+
+interface OrderSalesRepresentative
+{
+    /**
+     * Return the sales representative id for storing.
+     *
+     * @return int
+     */
+    public function getSellerId(): int;
```
Comment (viezel) [inline src/Orders/OrderSalesRepresentative.php]: refactor please. `GetSalesRepId()`

**Pull request merged**